MultiSource/Benchmarks/Olden/bh/walksub.c

```diff
@@ -24,7 +24,7 @@ hgstruct walksub(nodeptr p, real dsq, real tolsq, hgstruct hg, int level)
 {
   register int k, i;
   register nodeptr r = NULL;
-  nodeptr tmp _Checked[NSUB];
+  nodeptr tmp _Checked[NSUB] = {0};
```

Comment:
Is tmp used in the body of the function?   If it isn't, it should be deleted.

Reply:
Yes it is. Here is the code tmp appears:
hgstruct walksub(nodeptr p, real dsq, real tolsq, hgstruct hg, int level)
{
  register int k, i;
  register nodeptr r = NULL;
  nodeptr tmp _Checked[NSUB] = {0};
...